docs/components/dialogs.md

```diff
@@ -0,0 +1,471 @@
+<!--docs:
+title: "Dialogs"
+layout: detail
+section: components
+excerpt: "Dialogs are modal windows that require interaction."
+iconId: 
+path: /catalog/dialog/
+-->
+
+
+# Dialogs
+
+Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.
+
+There are four types of dialogs:
+
+1. [Alert](#alert-dialog)
+1. [Simple](#simple-dialog)
+1. [Confirmation](#confirmation-dialog)
+1. [Full-screen](#full-screen-dialog)
+
+<br>
+
+1. <img src="assets/dialogs/alert-dialog.png" alt="Alert dialog: discard" width=50%>
+1. <img src="assets/dialogs/simple-dialog.png" alt="Simple dialog: selection" width=50%>
+1. <img src="assets/dialogs/confirmation-dialog.png" alt="Confirmation dialog: selection confirmation" width=50%>
+1. <img src="assets/dialogs/full-screen-dialog.png" alt="Full-screen dialog: event" width=50%>
+
+## Contents
+
+- [Using banners](#using-dialogs)
+- [Alert](#alert-dialog)
+- [Simple](#simple-dialog)
+- [Confirmation](#confirmation-dialog)
+- [Full-screen](#full-screen-dialog)
+- [Dialog theming](#dialog-theming)
+
+## Using dialogs
+
+A dialog is a type of modal window that appears in front of app content to provide critical information or ask for a decision. Dialogs disable all app functionality when they appear, and remain on screen until confirmed, dismissed, or a required action has been taken.
+
+Dialogs are purposefully interruptive, so they should be used sparingly.
+
+Before you can use Material dialogss, you need to import the Material Components package for Flutter: `package:flutter/material.dart`.
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
```

Comment:
"You need to be using a..." -> "You need to use..."

Reply:
done